### PR TITLE
feat: Add a daemon thread to ensure that it will not exit abnormally

### PR DESCRIPTION
**Pull request by complone**
Daemon thread: ensure that the three threads of EventBusListener, EventRuleTransfer and EventTargetPusher will not exit abnormally; at the same time, exit normally and receive the signal of Kill-15 (otherwise the virtual machine cannot exit normally)

### CODE REVIEW

**Comment by complone** [comment]
Since PR#94 handles conflicts, open a new PR to handle

File: adapter/runtime/src/main/java/org/apache/rocketmq/eventbridge/adapter/runtime/boot/EventRuleTransfer.java [hunk 1]
```diff
@@ -75,15 +77,15 @@ public void run() {
...
-            Map<String, TransformEngine<ConnectRecord>> latestTransformMap = circulatorContext.getTaskTransformMap();
+            latestTransformMap = circulatorContext.getTaskTransformMap();
             if (MapUtils.isEmpty(latestTransformMap)) {
                 logger.warn("latest transform engine is empty, continue by curTime - {}", System.currentTimeMillis());
                 this.waitForRunning(3000);
                 continue;
             }
 
             List<ConnectRecord> afterTransformConnect = Lists.newArrayList();
-            List<CompletableFuture<Void>> completableFutures = Lists.newArrayList();
```
Comment (Jashinck) [inline adapter/runtime/src/main/java/org/apache/rocketmq/eventbridge/adapter/runtime/boot/EventRuleTransfer.java:86]: 此处局部变量，不需要全局化，主线程关闭后，资源无需手动释放

File: adapter/runtime/src/main/java/org/apache/rocketmq/eventbridge/adapter/runtime/boot/EventRuleTransfer.java [hunk 2]
```diff
@@ -75,15 +77,15 @@ public void run() {
                 this.waitForRunning(1000);
                 continue;
             }
-            Map<String, TransformEngine<ConnectRecord>> latestTransformMap = circulatorContext.getTaskTransformMap();
```
Comment (Jashinck) [inline adapter/runtime/src/main/java/org/apache/rocketmq/eventbridge/adapter/runtime/boot/EventRuleTransfer.java:78]: 变量，不需要全局化，资源释放参考下文CirculatorContext中：releaseTaskTransform方法，新建方法在CirculatorContext中完成相关资源释放。

File: adapter/runtime/src/main/java/org/apache/rocketmq/eventbridge/adapter/runtime/boot/EventRuleTransfer.java (hunk 1 above)
Comment (complone) [reply adapter/runtime/src/main/java/org/apache/rocketmq/eventbridge/adapter/runtime/boot/EventRuleTransfer.java]: done

File: adapter/runtime/src/main/java/org/apache/rocketmq/eventbridge/adapter/runtime/boot/EventRuleTransfer.java (hunk 2 above)
Comment (complone) [reply adapter/runtime/src/main/java/org/apache/rocketmq/eventbridge/adapter/runtime/boot/EventRuleTransfer.java]: done

File: adapter/runtime/src/main/java/org/apache/rocketmq/eventbridge/adapter/runtime/Runtime.java [hunk 3]
```diff
@@ -58,17 +64,40 @@ public class Runtime {
     private ErrorHandler errorHandler;
 
     @PostConstruct
-    public void initAndStart() {
-        logger.info("Start init runtime.");
+    public void initAndStart() throws Exception {
+        logger.info("Start init runtimer.");
```
Comment (Jashinck) [inline adapter/runtime/src/main/java/org/apache/rocketmq/eventbridge/adapter/runtime/Runtime.java:68]: runtimer -> runtime

File: adapter/runtime/src/main/java/org/apache/rocketmq/eventbridge/adapter/runtime/boot/EventRuleTransfer.java [hunk 4]
```diff
@@ -117,4 +117,18 @@ public void run() {
...
+    public void start() {
+        thread.start();
+    }
+
+    @Override
+    public void shutdown() {
+        try {
+            circulatorContext.releaseTaskTransform();
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (Jashinck) [inline adapter/runtime/src/main/java/org/apache/rocketmq/eventbridge/adapter/runtime/boot/EventRuleTransfer.java:130]: 打印具体业务领域及其堆栈信息，不建议直接打印堆栈信息。下同（Transfer, Trigger）

**Review by Jashinck — APPROVED** [review]

**Review by 2011shenlin — APPROVED** [review]

**Pull request merged**